types/index.d.ts

```diff
@@ -116,7 +116,7 @@ declare namespace Choices {
     containerInner?: () => HTMLElement;
     itemList?: () => HTMLElement;
     placeholder?: (value: string) => HTMLElement;
-    item?: (data: any) => HTMLElement;
+    item?: (globalClasses: any, data: any, removeItemButton: boolean) => HTMLElement;
```

Comment:
The majority of these look out of date from what I can see

Reply:
PR submitted for updating these types #619